lib/spec/router.sw-spec.js

```diff
@@ -206,4 +206,142 @@ describe('Router instances', function () {
     });
   });
 
+
+  describe('Matching algorithm', function() {
+    describe('match()', function() {
+      var mw = function() {};
+
+      it('should match universal registrations (*)', function() {
```

Comment:
Let's use the same vocabulary `should match anonymous placeholders`. Does it make sense to you?